CONTRIBUTING.md

```diff
@@ -33,11 +33,23 @@ The easiest workflow is to work through Foundry, and follow the following steps:
 2. Fill out the details, modifiers, etc.
 3. Right-clicking the item in the Items sidebar tab and select Export Data.
 4. Save the JSON file into the appropriate compendium subfolder.
+    * Make sure the name of the JSON follows these simple rules
```

Comment:
I'm not sure the filename is the important part. It's the "Name" value inside the file which should be like the book. All of the json is renamed by cook, as far as I know.

Reply:
Okay, had a different impression on how that works. Reflected the statements from above in another commit